Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
```

Comment:
Using a simple password like '1234' for the root user is a security risk. Consider using a more secure password.